### PR TITLE
Swith to new instance types endpoint

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Uses the new static endpoint to load instance types.

Depends on https://github.com/RHEnVision/provisioning-backend/pull/245

### CODE REVIEW

File: src/Components/InstanceTypesSelect/index.js [hunk 1]
```diff
@@ -53,11 +44,11 @@ const InstanceTypesSelect = ({ setValidation }) => {
...
         key={index}
-        description={`${instanceType.cores} cores | 
-          ${instanceType.vcpus} vCPU | 
+        description={`${instanceType.cores} cores |
+          ${instanceType.vcpus} vCPU |
           ${(parseFloat(instanceType.memory_mib) / 1024).toFixed(
             1
-          )} GiB memory | 
+          )} GiB memory |
           ${
```
[the remark this answers is not in the document]
Comment (ezr-ondrej) [reply src/Components/InstanceTypesSelect/index.js:52]: I'd probably do this separatelly and flag unsupported as disabled - unable to be selected
(Why later: I'd like to keep using t1.micro as long as possible)

[the remark this answers is not in the document]
Comment (lzap) [reply src/Components/InstanceTypesSelect/index.js:52]: Sure.

**Comment by akhil-jha** [comment]
I build the env with this PR. The happy path works, but I do see something wrong while selecting the instance type. Sending the screenshot on slack.

**Comment by lzap** [comment]
You can merge this PR, I will fix the double types, backend error.

**Review by akhil-jha — APPROVED** [review]

**Comment by lzap** [comment]
https://github.com/RHEnVision/provisioning-backend/pull/260

**Pull request merged**